src/themes.ts

```diff
@@ -125,8 +125,9 @@ export const light: PreciseFullTheme = {
   accordionLine: `1px solid ${colors.lighterGray}`,
   accordionPadding: `${distance.medium}`,
   accordionContentPadding: `${distance.small} ${distance.medium} ${distance.xlarge} ${remCalc('50px')}`,
-  actionLinkHoverBackground: colors.skyBlue,
-  actionLinkDisabledText: colors.white,
+  actionLinkHoverBackground: colors.ocean,
+  actionLinkFocusBackground: colors.midnight,
```

Comment:
midnight is ui7 - ocean and pinkSwan I don't know. afaik they are not on the theme; so either this is wrongly specified in the pattern library or missing from the color spec.

Reply:
ui6 was changed from transparent to ocean
we looked for all appearances of ui6 inside pharos and renamed it to transparent
we removed actionLinkHoverBackground and actionLinkFocusBackground from themes